concourse/pipelines/linux-image-build.jsonnet

```diff
@@ -239,6 +239,8 @@ local imgpublishjob = {
   trigger:: if tl.env == 'testing' then true
   else false,
 
+  citfilter:: '^(%s)$' % self.citsuites,
```

Comment:
Why define `citsuites` just to interpolate it into `citfilter`? Just allow the caller to specify the full regexp.

Reply:
Oh, I see, to allow people to add suites. If that's the approach, `citsuites` should probably be a list that gets concatenated with `|` characters when it's put into the surrounding `^()$` so that the usage is clearer.

Reply:
I think expecting the user to pass in the full regexp might be better here so we're not messing with transforming input.